sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -240,6 +241,29 @@ def resources(self, value: Union[Dict, JobResourceConfiguration]):
             value = JobResourceConfiguration(**value)
         self._resources = value
 
+    @property
```

Comment:
please add some test for this change

Reply:
added in dsl/unittests/test_command_builder.py